src/tasks/installers/operator.ts

```diff
@@ -171,6 +175,8 @@ export class OperatorTasks {
 
   updateTasks(flags: any, command: Command): Listr {
```

Comment:
In updateTasks we have to be compatible with previous versions.
1. Check if resource with namespace prefix exists and update it if so
2. Otherwise, check if resource with a default name exists and update it if so